internal/alert/provider/email/send.go

```diff
@@ -0,0 +1,151 @@
+package email
+
+import (
+	"bytes"
+	"crypto/tls"
+	"fmt"
+	"math/rand"
+	"mime"
+	"mime/multipart"
+	"mime/quotedprintable"
+	"net"
+	"net/mail"
+	"net/smtp"
+	"net/textproto"
+	"strings"
+	"time"
+
+	"github.com/balerter/balerter/internal/alert/message"
+	"github.com/pkg/errors"
+	"go.uber.org/zap"
+)
+
+func (e *Email) Send(message *message.Message) error {
+	var (
+		c    *smtp.Client
+		conn net.Conn
+		err  error
+	)
+	if e.conf.ServerPort == "465" {
```

Comment:
If will be use a different port, it may be worth using the configuration flag cfg.IsSecure instead checking by port value?

Reply:
Added RequireTLS config.

Reply:
Maybe, change condition `e.conf.ServerPort == "465"` to `e.conf.RequireTLS`?

Reply:
Let me quote Wikipedia:

"Some email service providers allow their customers to use the SMTPS protocol to access a TLS-encrypted version of the "submission" service on port 465. This is a different service from what the original IANA-registration dedicated the port to.

RFC 8314 aims to rectify this problem and integrate the use of port 465 as a TLS-encrypted "submission" port into the well-known port registrations published by IANA. The proposed service name is "submissions"."

Reply:
I mean, but what if the client uses a different port for various reasons? I try to avoid such constants in the code)

Reply:
Yeah me too but for 465 it makes sense. For the other ports I added e.conf.RequireTLS see my commit here:
https://github.com/balerter/balerter/pull/6/commits/f015ca38ae2d50fedd8bc92a7176d2801b24574c

Reply:
Ok :+1: